api/tests/acceptance/application/authentication-route_test.js

```diff
@@ -2,6 +2,7 @@ const querystring = require('querystring');
 const { expect, databaseBuilder, knex } = require('../../test-helper');
 const tokenService = require('../../../lib/domain/services/token-service');
 const AuthenticationMethod = require('../../../lib/domain/models/AuthenticationMethod');
+const settings = require('../../../lib/config');
```

Comment:
On peut limiter le scope à importer
```suggestion
const { rateLimit: settings } = require('../../../lib/config');
```

Reply:
il faut pouvoir restaurer le rate limit après les tests, donc non.

Reply:
https://github.com/1024pix/pix/pull/4416#discussion_r902643569